src/gash/router/server/messages/wrk_messages/handlers/ElectionMessageHandler.java

```diff
@@ -26,7 +24,6 @@ public void handleMessage(WorkMessage workMessage, Channel channel) {
 			return;
```

Comment:
We need to check if this message is for current handler before checking nexthandler with null.